<?xml version="1.0" encoding="utf-8"?>
<TestRun id="640b0dd0-b657-4f9c-bb07-cac32466863f" name="MilleB@SYVE-20190118MB 2020-04-21 23:04:30" runUser="SYVE\MilleB" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2020-04-21T23:04:30.4842992+02:00" queuing="2020-04-21T23:04:30.4843008+02:00" start="2020-04-21T23:04:28.8693681+02:00" finish="2020-04-21T23:04:30.5009865+02:00" />
  <TestSettings name="default" id="20e5101b-22a5-4aff-9f22-4fa4acedcce3">
    <Deployment runDeploymentRoot="MilleB_SYVE-20190118MB_2020-04-21_23_04_30" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="10dc1090-76f5-43e3-8293-3868428ac287" testId="51726c4d-172d-7d8b-abde-8696baa37e09" testName="XUnitTestProject.UnitTest1.Test1" computerName="SYVE-20190118MB" duration="00:00:00.0040297" startTime="2020-04-21T23:04:30.3143453+02:00" endTime="2020-04-21T23:04:30.3143486+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="10dc1090-76f5-43e3-8293-3868428ac287">
      <Output>
        <ErrorInfo>
          <Message>Assert.Equal() Failure&#xD;
Expected: 2&#xD;
Actual:   1</Message>
          <StackTrace>   at XUnitTestProject.UnitTest1.Test1() in C:\Users\MilleB.SYVE\source\repos\XUnitTestProject\XUnitTestProject\UnitTest1.cs:line 13</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="9b76a1b8-4edd-40bb-af25-344aa83d8956" testId="6ee766df-7287-8119-1394-64b94c186811" testName="XUnitTestProject.UnitTest1.Test2" computerName="SYVE-20190118MB" duration="00:00:00.0051339" startTime="2020-04-21T23:04:30.2856062+02:00" endTime="2020-04-21T23:04:30.2857117+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="9b76a1b8-4edd-40bb-af25-344aa83d8956" />
  </Results>
  <TestDefinitions>
    <UnitTest name="XUnitTestProject.UnitTest1.Test1" storage="c:\users\milleb.syve\desktop\testura.mutation\tests\testura.mutation.tests\testrunner\samples\xunit\failingtests\xunittestproject.dll" id="51726c4d-172d-7d8b-abde-8696baa37e09">
      <Execution id="10dc1090-76f5-43e3-8293-3868428ac287" />
      <TestMethod codeBase="C:\Users\MilleB.SYVE\Desktop\Testura.Mutation\tests\Testura.Mutation.Tests\TestRunner\Samples\Xunit\FailingTests\XUnitTestProject.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="XUnitTestProject.UnitTest1" name="Test1" />
    </UnitTest>
    <UnitTest name="XUnitTestProject.UnitTest1.Test2" storage="c:\users\milleb.syve\desktop\testura.mutation\tests\testura.mutation.tests\testrunner\samples\xunit\failingtests\xunittestproject.dll" id="6ee766df-7287-8119-1394-64b94c186811">
      <Execution id="9b76a1b8-4edd-40bb-af25-344aa83d8956" />
      <TestMethod codeBase="C:\Users\MilleB.SYVE\Desktop\Testura.Mutation\tests\Testura.Mutation.Tests\TestRunner\Samples\Xunit\FailingTests\XUnitTestProject.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="XUnitTestProject.UnitTest1" name="Test2" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="51726c4d-172d-7d8b-abde-8696baa37e09" executionId="10dc1090-76f5-43e3-8293-3868428ac287" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="6ee766df-7287-8119-1394-64b94c186811" executionId="9b76a1b8-4edd-40bb-af25-344aa83d8956" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Failed">
    <Counters total="2" executed="2" passed="1" failed="1" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.0 (64-bit .NET Core 3.1.3)&#xD;
[xUnit.net 00:00:00.48]   Discovering: XUnitTestProject&#xD;
[xUnit.net 00:00:00.53]   Discovered:  XUnitTestProject&#xD;
[xUnit.net 00:00:00.54]   Starting:    XUnitTestProject&#xD;
[xUnit.net 00:00:00.65]       Assert.Equal() Failure&#xD;
[xUnit.net 00:00:00.65]       Expected: 2&#xD;
[xUnit.net 00:00:00.65]       Actual:   1&#xD;
[xUnit.net 00:00:00.65]       Stack Trace:&#xD;
[xUnit.net 00:00:00.65]         C:\Users\MilleB.SYVE\source\repos\XUnitTestProject\XUnitTestProject\UnitTest1.cs(13,0): at XUnitTestProject.UnitTest1.Test1()&#xD;
[xUnit.net 00:00:00.65]   Finished:    XUnitTestProject&#xD;
</StdOut>
    </Output>
    <RunInfos>
      <RunInfo computerName="SYVE-20190118MB" outcome="Error" timestamp="2020-04-21T23:04:30.3104674+02:00">
        <Text>[xUnit.net 00:00:00.64]     XUnitTestProject.UnitTest1.Test1 [FAIL]</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
</TestRun>